collections/guides/setup_first_reactor/setup_first_reactor.md

```diff
@@ -49,11 +49,11 @@ That's it! Once you've saved your first Reactor, you can begin interacting with
 
 ## Use Your New Reactor
 
-With your Reactor created, it's time to start sending your tokens into it and generate your first reaction. Depending on which Reactor you chose to create, you'll need to create the correct corresponding token type (for example the "Parrot" reactor is for [Atomic Cards](https://docs.basistheory.com/api-reference/#atomic-cards)).
+With your Reactor created, it's time to start sending your tokens into it and generate your first reaction. Depending on which Reactor you chose to create, you'll need to pass in the corresponding request parameters it needs. 
 
-Now that you have a token, we can use the [/react](https://docs.basistheory.com/api-reference/#atomic-cards-react-with-an-atomic-card) endpoint to send the token to your new Reactor.
+Request parameters are passed in via the `args` property object which may contain simple values or complex objects. You could also include token interpolations if you want to use an existing encrypted token's data to pass as parameters to your reactor. Check the [/react](https://docs.basistheory.com/api-reference/#reactors-invoke-a-reactor) endpoint's documentation for more details. 
```

Comment:
`interpolations` -> `interpolation patterns`?

Reply:
It would also be relevant to link to the new guide I created - `[Use Token Data in Reactors](/guides/use-token-data-in-reactors/)`